modules/ROOT/partials/misc/autocorrect-options-notes.adoc

```diff
@@ -0,0 +1,3 @@
+NOTE: The user’s preference takes precedence over the configuration at the next initialization. The user’s preference is saved to `localStorage` when the user enables or disables the autocorrection via the menu item.
+
+NOTE: When used with the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] plugin, autocorrection is disabled when a non-English language is selected. A user can, however, re-enable autocorrection by selecting the *Tools > Autocorrect* menu item.
```

Comment:
```suggestion
NOTE: When used with the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] plugin, autocorrect and capitalization capabilities are disabled when a non-English language is selected.  After being disabled, they will not be automatically re-enabled when a user selects one of the English languages. However, the user can re-enable them in the *Tools* menu.
```

@brian-forte , just want to provide some info. Autocorrect and Capitalization menu items are added to the *Tools* menu when autocorrect plugin is added to the tinymce. When a non-English is selected, both are disabled and will not be automatically re-enabled even if users select an English language afterward.

Reply:
Added this to the *Known issues* section of `autocorrect.adoc`.